### PR TITLE
Version updates, readme fixes, manifest fixes, env. for argo's container runtime executor and bug fixes for Okteto dev. env. setup.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
## Proposed changes

* Version updates
* README fixes
* Manifest fixes
* Container runtime executor env. var. for argo workflow controller.
* Okteto dev. env. setup enhancements.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml [hunk 1]
```diff
@@ -168,6 +164,18 @@ spec:
               value: "litmuschaos/chaos-runner:1.13.3"
             - name: LITMUS_CHAOS_EXPORTER_IMAGE
               value: "litmuschaos/chaos-exporter:1.13.3"
+            - name: SERVER_SERVICE_NAME
```
Comment (rajdas98) [inline docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: For the namespace, we don't need these details. User have to set the endpoint manually in the env- "PORTAL_ENDPOINT"

File: litmus-portal/namespaced-k8s-template.yml [hunk 2]
```diff
@@ -126,10 +126,6 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            - name: PORTAL_SCOPE
-              value: "namespace"
-            - name: PORTAL_ENDPOINT
```
Comment (rajdas98) [inline litmus-portal/namespaced-k8s-template.yml:131]: We need this env for namespace mode

**Review by rajdas98 — CHANGES_REQUESTED** [review]

File: litmus-portal/namespaced-k8s-template.yml (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/namespaced-k8s-template.yml:131]: But why do we need this exactly?

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml (hunk 1 above)
Comment (ishangupta-ds) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: In namespace mode why do user need to hardcode this PORTAL_ENDPOINT?

File: litmus-portal/namespaced-k8s-template.yml (hunk 2 above)
Comment (rajdas98) [reply litmus-portal/namespaced-k8s-template.yml:131]: For namespace mode we ask PORTAL_ENDPOINT

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml (hunk 1 above)
Comment (rajdas98) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: Because namespace mode RBAC need cluster level permission

Comment (ishangupta-ds) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: I have tried this out in namespace mode and it worked fine also on a very restricted namespace mode of Okteto

Comment (rajdas98) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: Have you tried to connect an agent using namespace mode? 

I think we are retrieving the node details which requires cluster level permission https://github.com/litmuschaos/litmus/blob/879b9cf303706bfffd7a366342eea7579dfb8a25/litmus-portal/graphql-server/pkg/k8s/cluster.go#L195

Comment (ishangupta-ds) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: Will test on all platforms and revert back.

File: litmus-portal/namespaced-k8s-template.yml (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/namespaced-k8s-template.yml:131]: This may not be required.

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml (hunk 1 above)
Comment (ishangupta-ds) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:167]: yes @rajdas98 i am testing it for all cases, will update the PR accordingly if needed.

File: litmus-portal/namespaced-k8s-template.yml (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/namespaced-k8s-template.yml:131]: Reverted this change.

**Review by rberrelleza — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**